Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -9,24 +9,44 @@ import BraintreeCore
 
     // MARK: - Public Properties
 
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    public var userAuthenticationEmail: String?
+
+    // MARK: - Internal Properties
+
     /// Optional: Used to determine if the customer will use the PayPal app switch flow.
     /// Defaults to `false`.
     /// - Note: This property is currently in beta and may change or be removed in future releases.
-    public var enablePayPalAppSwitch: Bool
+    var enablePayPalAppSwitch: Bool = false
 
-    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
-    public var userAuthenticationEmail: String?
+    /// The URL to use for the PayPal app switch flow. Must be a valid HTTPS URL dedicated to Braintree app switch returns.
+    var universalLink: URL?
 
-    // MARK: - Initializer
+    // MARK: - Initializers
+
+    /// Initializes a PayPal Vault request
+    /// - Note: This initializer should be used for merchants using the PayPal App Switch flow
+    /// - Parameters:
+    ///   - userAuthenticationEmail: Required: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    ///   - enablePayPalAppSwitch: Required: Used to determine if the customer will use the PayPal app switch flow.
+    ///   This property is currently in beta and may change or be removed in future releases.
```

Comment:
nit(optional): can we move this line after all the parameter comments

Reply:
I'm actually going to move it with the main note: https://github.com/braintree/braintree_ios/pull/1232/commits/631b3e30ff403465db9d203588194de35cfbdc71. Here is what a merchant will see:
![Screenshot 2024-03-26 at 12 15 36 PM](https://github.com/braintree/braintree_ios/assets/20733831/53deeadb-64a4-4b69-8fdb-23980b08a037)